docs/integrations/immer-middleware.md

```diff
@@ -19,6 +19,8 @@ npm install immer
 
 ## Usage
 
+(Note the extra parenthesis after the type parameter as mentioned in the [Typescript Guide](.../guides/typescript.md)).
```

Comment:
The plural form of _parenthesis_ is _parentheses_, [as per Merriam-Webster Dictionary](https://www.merriam-webster.com/dictionary/parenthesis).

```suggestion
(Note the extra parentheses after the type parameter as mentioned in the [Typescript Guide](.../guides/typescript.md)).
```

Reply:
Ah, good point, fixed here and in typescript.md.